portality/crosswalks/article_crossref_xml.py

```diff
@@ -0,0 +1,269 @@
+import os
```

Comment:
Can we add an explicit crosswalk test in `test_xwalk.py`.

We also need to run the results of the xwalk by the DOAJ team.

Reply:
There is no explicit crosswalk for doaj schema in `test_xwalk.py` as well. Do you think it would be worth adding alongside Crossref one?